OrbitLinuxTracing/TestPuppet.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This is test executable used by LinuxTracing tests to test profiling functionality
+// The control is done by executing commands
+
+#include <dlfcn.h>
+
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+
+// dlopen and call function_that_sleeps_for_one_second 10 times.
+static void run_dlopen() {
+  static const char* kLibraryFile = "libOrbitLinuxTracingTestPuppetLibrary.so";
+  static const char* kFunctionName = "function_that_works_for_considerable_amount_of_time";
+  // Setting rpath in CMAKE is a nightmare.. I was not able to do that, so we are going
+  // to emulate "$ORIGIN/../lib" rpath here.
```

Comment:
So what are
```
set(CMAKE_SKIP_BUILD_RPATH FALSE)
set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
set(CMAKE_INSTALL_RPATH $ORIGIN)
```
in the `CMakeLists.txt` file for? Maybe add a comment to those as well?